src/app/containers/Byline/index.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useContext } from 'react';
+import Byline from '@bbc/psammead-byline';
+import { ServiceContext } from '#contexts/ServiceContext';
+import bylineBlockPropTypes from '#models/propTypes/byline';
+import { GridItemConstrainedMedium } from '#lib/styledGrid';
+
+const BylineContainer = ({ blocks, className }) => {
+  const { service } = useContext(ServiceContext);
+
+  if (!blocks) {
+    return null;
+  }
+
+  const { name, title } = blocks[0];
+
+  if (name && title) {
+    return (
+      <GridItemConstrainedMedium className={className}>
```

Comment:
I took borrowed the use of `GridItemConstrainedMedium` from `ArticleTimestamp`: https://github.com/bbc/simorgh/blob/latest/src/app/containers/ArticleTimestamp/index.jsx#L58..L70

Is there any documentation for the items in https://github.com/bbc/simorgh/blob/latest/src/app/lib/styledGrid ? I was just lucky to find a suitable example, I'm not sure how I would have chosen a style otherwise?

Reply:
I think the only thing we have are stories on storybook as far as I know https://bbc.github.io/simorgh/?path=/story/grid-examples--griditemconstrainedsmall

Reply:
I'm not super familiar with the grid implementation. Would you be able to briefly explain why we need a `className` prop here?

Reply:
So using `className` allows the component to have additional styles applied to it here: https://github.com/bbc/simorgh/pull/5512/files#diff-fdf1e6ecf0538604611b254a4f30cfd4R100-R106 

This is implementing the pattern described here: https://styled-components.com/docs/advanced#styling-normal-react-components 

I don't know if there is a better way of doing this but I saw the same pattern used for the `Timestamp`,